src/trackinfoobject.cpp

```diff
@@ -783,13 +783,14 @@ bool TrackInfoObject::isDirty() {
 
 int TrackInfoObject::getRating() const {
     QMutexLocker lock(&m_qMutex);
-    return m_iRating;
+    return m_metadata.getRating();
```

Comment:
How does Windows media player behave if there is an embedded rating with a foreign email field? 

Banshee ignored the rating if I change "Banshee" to "Mixxx  ", which is correct in the first place, but is not what the user might expect. He might want to use the ID3 tag to see or even synchronize ratings among several media players.  

In my personal use case, Mixxx is not my main media player. I use Clementine together with my family.  
It would be nice, If I be able to see the Clementine ratings in Mixxx but in a separate column, without destroying the DJ rating. 

An other DJ has a second mixing app along with Mixxx (I can't think of a reason why) he might want to keep the ratings in sync. If we want to do that, I think there is now way around exposing the Email field to the user.